internal/sync/syncer.go

```diff
@@ -267,7 +270,7 @@ func (r *resourceInfoProviderStub) IsNamespaced(_ schema.GroupKind) (bool, error
 	return false, nil
 }
 
-// sync compares the live state of the watched resources to target state defined in git
+// sync compares the live state of the watched includedResources to target state defined in git
```

Comment:
```suggestion
// sync compares the live state of the watched resources to target state defined in git
```

Reply:
Oops sorry for these unintended changes. Thanks for catching it!